app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,7 @@
 
     <application
         android:allowBackup="true"
+        android:configChanges="orientation|screenSize"
```

Comment:
Do you really need this?

Reply:
Removed unnecessary config in AndroidManifest.